readme.md

```diff
@@ -38,39 +38,39 @@ Hello, Node!
 
 * **Question:** Imagine you have a file called `classmates.txt`. For each part of CRUD, describe how the action would interact with the file.
 
-* **Your Answer:**
+* **Your Answer:** Create the `classmates.txt` file, Read what's inside the `classmates.txt` file, Update the contents of the `classmates.txt` file like adding or removing someone, Delete the `classmates.txt` file.
 
 ---
 
 - [ ] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
 
 * **Question:** What methods represent each CRUD action?
 
-* **Your Answer:**
+* **Your Answer:** Create is likely `fs.writeFile()` to create a new file, Read is `fs.readFile()` to read a file in, Update is `fs.appendFile()`, Remove is `fs.unlink()`
 
 ---
 
 - [ ] Take a look at the following two methods: [fs.readFile()](https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback) and [fs.readFileSync()](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options).
 
 * **Question:** What is the difference between these two methods?
 
-* **Your Answer:**
+* **Your Answer:** One version is asynchronous and the other is synchronous. The async function will not block the stack like the sync function will. Async will return a promise.
```

Comment:
Which is which? The asynchronous version will not actually return a promise, but will instead take a callback.

Reply:
`fs.readFile()` - asynchronous
`fs.readFileSync()` - synchronous
Sorry for not being more clear here.